### PR TITLE
Fixes #29: Don't show maven transfer progress by default

### LINKED ISSUE #29: Don't show maven transfer progress by default
## I expect

- don't show maven transfer progress by default

## Note

- maven transfer progress make super-sast.log unreadable
- unless it's explicitly enabled via a specific parameter, it should be set MAVEN_ARGS=" --no-transfer-progress ", or it could even be enabled resetting MAVEN_ARGS=""

### PR DESCRIPTION
**Pull request by nocturn9x**
# This PR
- Fixes #29 

# Note
- A specialized `--log-maven-progress` option has been added to instruct super-sast to dump the maven transfer progress to the log file

### CODE REVIEW

**Comment by nocturn9x** [comment]
@ioggstream

File: entrypoint.py [hunk 1]
```diff
@@ -336,6 +341,12 @@ def _copy_java_validators():
         default="",
         action="store_true",
     )
+    parser.add_argument(
```
Comment (ioggstream) [inline entrypoint.py:344]: this solution is ok for a maven specific script.
In this case, adding a parameter for each specific tool (maven, trivy, ...) will make the tool unmaintainable in time.

File: entrypoint.py [hunk 2]
```diff
@@ -205,7 +205,9 @@ def _localize(path, config_dir):
     return (config_dir / path).absolute().as_posix()
 
 
-def run_sast(tool, command, env, config_dir, log_file=stdout, run_all=True):
+def run_sast(
+    tool, command, env, config_dir, log_file=stdout, run_all=True, maven_logs=False
```
Comment (ioggstream) [inline entrypoint.py:209]: passing tool-specific variables in this function leads to an unmaintainable script with tens of tool-specific variables.

This solution would be ok for a maven-specific tool

**Review by ioggstream — CHANGES_REQUESTED** [review]
I won't add tool-specific variables. Instead the #29 issue suggests using a maven-specific env var: do you find any issue with the general idea detailed in #29? Please detail any shotcomings there.

**Comment by nocturn9x** [comment]
That looks like a misunderstanding on my part on the term "parameter". I'll work on the env var solution ASAP.

P.S. My bad!

**Comment by ioggstream** [comment]
> my bad ;)

no problem at all. Feel free to ask if the proposed solution seems weird or if you think that the code review is missing some important point.

**Comment by nocturn9x** [comment]
@ioggstream This should do it :)

**Pull request merged**